src/AWS/Lambda/Context.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE NamedFieldPuns, OverloadedStrings #-}
```

Comment:
Are you having to do this because of stylish-haskell? We have it enabled already in our package.yaml